Formula/p/protobuf-c.rb

```diff
@@ -24,38 +23,11 @@ class ProtobufC < Formula
     depends_on "libtool" => :build
   end
 
-  # TODO: `autoconf`, `automake`, and `libtool` are needed for the patches.
-  #       Remove when they are no longer needed.
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "libtool" => :build
   depends_on "pkg-config" => :build
   depends_on "protobuf"
 
-  # The next three patches are for compatibility with the newest protobuf.
-  # https://github.com/protobuf-c/protobuf-c/pull/556
-  # TODO: Uncomment `if build.head?` in `#install` when these are no longer needed.
-  patch do
-    url "https://github.com/protobuf-c/protobuf-c/commit/66a0b0d205224f63f19dd8f96abf9dcdc2112331.patch?full_index=1"
-    sha256 "a3561ad37f33048c59a1ceece246a515b62cef91126e4041056d10ea26a19230"
-  end
-
-  patch do
-    url "https://github.com/protobuf-c/protobuf-c/commit/7706c95d4835e75f182ab56d9dad5c8cd8517e0a.patch?full_index=1"
-    sha256 "86364b4da6e077bd9f89a82d6e2ac965776ee1a544e43d1964c2e800424cdb6e"
-  end
-
-  patch do
-    url "https://github.com/protobuf-c/protobuf-c/commit/66574f3fd85a205eb7c90b790477d5415364209e.patch?full_index=1"
-    sha256 "2d1d6edbd615dff4f0a9c4a974d325effefc44466e1855f8c0b88e5977962a9d"
-  end
-
   def install
-    # https://github.com/protocolbuffers/protobuf/issues/9947
-    ENV.append_to_cflags "-DNDEBUG"
```

Comment:
Might be safer to keep this, I think.

Reply:
fair enough, I will revert the change :)